### DISCUSSION TITLE
補眠了還是睏，有可能是社會性時差搞的鬼。

### DISCUSSION THREAD

**Discussion by dayu1004** [1101-GS4719-C/Python-Community-GS4719-C#555] [General]
「社會性時差」（social jetlag）是指我們的正常睡眠時間和上課或工作的時間不同。以週末假日來說，我們普遍會較平日還晚個幾小時起床，直到週一來臨，我們才又回復早起。假日和平日的作息差異，已經讓體內的生理時鐘像是穿梭在不同時區之間。

研究團隊發現一項有趣的事實：不論什麼年齡層的受測者，和二○○二年相比，二○一○年時都平均晚了二十分鐘就寢，但學校和工作時間卻沒有改變，表示這幾年間，社會性時差增加了。如果人們減少待在戶外的時間，身體較少接觸到自然光，就難以調整生理時鐘，讓自己早睡。

社會的作息也影響我們飲食的規律。在工作日，因為早起，吃早餐的時候身體可能還停在夜晚的狀態，所以在不對的時間進食，可能會干擾消化系統運作，這或許跟過胖有關。這項發現特別適用於青少年，因為他們自然的生理時鐘週期較長，比學校的作息還晚，因此青少年比其他年齡層的人更有社會性時差的問題。

過去發現輪班工作者常有健康風險，且工作及非工作時段的睡眠狀況差異很大。而這項研究結果顯示，不光是輪班工作這樣極端作息的族群，甚至平常週末及週間的睡眠差異，就足以影響健康了。
— — — — — — — — — — — — — — — — — — — — — — — — — — — — — — —
前陣子太忙的時候一直覺得自己怎麼睡都還是很累，後來才知道原來有社會時差這個東西的存在，才會補眠的時候睡越久反而更累。看完這篇又另外查了資料，發現大概平均日曬兩小時的話，可以調整約一小時的社會時差，因此能夠讓你在工作日起床時不那麼痛苦。
所以人類好像還是要跟大自然多接觸才可以變得更健康的樣子呢（？

**Comment by hapiwangy** [comment]
原來越睡越累還有這種可能，感謝分享

**Reply by 110409538** [reply]
冬天的時候,總是會賴床,然後就會越睡越累.

**Comment by ghost** [comment]
我只覺得現在怎麼睡都睡不飽QQ

**Comment by ThomasMeng0515** [comment]
第一次聽到社會性時差這個名詞，感謝分享!

**Comment by lx3211** [comment]
不能用美國跟我們睡眠時間完全相反來反駁爸媽ㄌ

**Comment by sean1325418** [comment]
簡單說就是沒事就多曬曬太陽有益身體健康

**Comment by cordydeng** [comment]
如果能讓平日和假日的作息不要有太大的差異 , 應該就能降低社會性時差的影響

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我知道甚麼是社會性時差。

**Comment by Sazmafuji** [comment]
難怪有時睡太久會更累

**Comment by yi-1111** [comment]
難怪最近覺得越睡越累

**Comment by HuanYuuuuuu** [comment]
最重要的是每天都要有固定的作息!

**Comment by bloodtrail0511** [comment]
我假日跟平日怎麼都差不多，都睡到中午XD

**Comment by HaoFish123** [comment]
謝謝分享，蠻有感觸的。

**Comment by Jay-722** [comment]
看來為了早睡早起，要常常去曬太陽了

**Comment by PoooYen** [comment]
難怪!!!! 原來˙我之前越睡越累就是這個原因

**Comment by d26488399** [comment]
看來假日早起也很重要

**Comment by Yuhsin000** [comment]
好想一直睡覺zzz，掌握自己的睡眠週期(大約90分鐘)，也可以睡醒時比較不累喔

**Comment by Lai27330** [comment]
謝謝分享，難怪有時總覺得明明同樣睡到8個小時了，精神卻沒有比較好，也會降低思考能力，原來是社會性時差的問題啊，感覺了解睡眠機制後對於工作效率會有很大的幫助呢!

**Comment by stan890425** [comment]
曬太陽很舒服ㄟ~~

**Comment by sys511613** [comment]
真的常常為了配合各種事情調整自己的睡眠時間，導致身體跟社會不同調，原來這個叫社會性時差，感謝分享

**Comment by jcpan821** [comment]
早睡早起還是比較好的

**Comment by alexthebigpp** [comment]
原來如此，一定不是我每天都兩三點才睡搞的鬼

**Comment by yenwei12138** [comment]
真的年紀越大越晚睡，又容易累，看來這就是成長吧(誤

**Comment by etin2468** [comment]
我覺得我生理時鐘真的完全壞掉了

**Comment by Lynn12378** [comment]
居然有這樣的事!感謝樓主分享，看來真的要多曬太陽了QQ

**Comment by Jimmy911218** [comment]
確實現在社會的節奏使人越來越少在陽光下了~除非字跡想出去去
